FixBookUI.java

```diff
@@ -1,88 +1,70 @@
 import java.util.Scanner;
 
-
 public class FixBookUI {
+	public static enum FixBookUiState {
+		INITIALISED, READY, FIXING, COMPLETED
+	};
 
-	public static enum UI_STATE { INITIALISED, READY, FIXING, COMPLETED };
-
-	private FixBookControl CoNtRoL;
-	private Scanner input;
-	private UI_STATE StAtE;
+	private FixBookControl fixBookControl;
+	private Scanner keybordInput;

```

Comment:
@GoJoeyGo keybordInput into keyboardInput. Thankyou

Reply:
will change keybordInput to keyboardInput i think i did it for all of the code

Reply:
heheh yes but not a big deal